### PR TITLE
Replaced GTM with RudderStack

### PR DESCRIPTION
**Pull request by danthebaker**

### CODE REVIEW

File: doc/overrides/main.html [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
 {% block extrahead %}
-  <!-- Google Tag Manager -->
-  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','GTM-PGZQGKM');</script>
-  <!-- End Google Tag Manager -->
+  <script> 
+    rudderanalytics=window.rudderanalytics=[];for(var methods=["load","page","track","identify","alias","group","ready","reset","getAnonymousId","setAnonymousId"],i=0;i<methods.length;i++){var method=methods[i];rudderanalytics[method]=function(a){return function(){rudderanalytics.push([a].concat(Array.prototype.slice.call(arguments)))}}(method)}rudderanalytics.load("1iJtD7UiXR23JlGPtE8YnzQH3KG","https://pachyderm-dataplane.rudderstack.com"),rudderanalytics.page();
```
Comment (jmcriffey) [inline doc/overrides/main.html:5]: This looks like it has the "Docs" write key. Is that intended, or does it need a different one?

File: doc/overrides/partials/integrations/analytics.html [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
 {% set analytics = config.google_analytics %}
-  <!-- Google Tag Manager -->
-  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','GTM-PGZQGKM');</script>
-  <!-- End Google Tag Manager -->
+<script> 
+  rudderanalytics=window.rudderanalytics=[];for(var methods=["load","page","track","identify","alias","group","ready","reset","getAnonymousId","setAnonymousId"],i=0;i<methods.length;i++){var method=methods[i];rudderanalytics[method]=function(a){return function(){rudderanalytics.push([a].concat(Array.prototype.slice.call(arguments)))}}(method)}rudderanalytics.load("1iJtD7UiXR23JlGPtE8YnzQH3KG","https://pachyderm-dataplane.rudderstack.com"),rudderanalytics.page();
```
Comment (jmcriffey) [inline doc/overrides/partials/integrations/analytics.html:3]: Same question here as above.

**Review by jmcriffey — COMMENTED** [review]
Looking good. I just had one question about the write keys.

File: doc/overrides/main.html (hunk 1 above)
Comment (danthebaker) [reply doc/overrides/main.html:5]: Yes, that's the intention. I should've mentioned that this was the docs site integration.

**Pull request merged**